src/main/java/frc/robot/subsystems/arm/ArmStageOne.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.InvertType;
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.ArmConstants;
+
+public class ArmStageOne extends SubsystemBase {
+  private TalonSRX armMotorPrimary;
+  private TalonSRX armMotorSecondary;
+  private DutyCycleEncoder magEncoder;
+  private double encoderOffset = ArmConstants.kStageOne_AbsEncoderInitialOffset;
+  private PIDController pid = new PIDController(ArmConstants.stageOne_kP,ArmConstants.stageOne_kI,ArmConstants.stageOne_kD);
+  private double setpointRad = encoderOffset;
+
+  /** Creates a new ArmStageOne. */
+  public ArmStageOne() {
+    armMotorPrimary = new TalonSRX(ArmConstants.kStageOne_MotorLeftID); //left is master
+    armMotorSecondary = new TalonSRX(ArmConstants.kStageOne_MotorRightID);
+
+    armMotorPrimary.configFactoryDefault();
+    armMotorSecondary.configFactoryDefault();
+
+    armMotorPrimary.follow(armMotorSecondary);
+    armMotorPrimary.setInverted(InvertType.OpposeMaster);
+
+    armMotorPrimary.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0,30);
+
+    armMotorPrimary.config_kP(0,ArmConstants.stageOne_kP,30);
+    armMotorPrimary.config_kI(0,ArmConstants.stageOne_kI,30);
+    armMotorPrimary.config_kD(0,ArmConstants.stageOne_kD,30);
+
+    armMotorPrimary.configClosedLoopPeriod(0, 1, 30);
+
+    //magEncoder = new DutyCycleEncoder(ArmConstants.kStageOne_MagEncoderID);
+
+    magEncoder.setDistancePerRotation(1);
+
+  }
+
+  public double getAbsoluteEncoderRad() {
```

Comment:
should just get position from the talonSRX selected feedback sensor

Reply:
also its not absolute

Reply:
So we shouldn't apply the offset?